### ISSUE TITLE
python-xgboost-git package is not recognized

### ISSUE THREAD

**Issue by mizabrik** [aurapm/aura#478]
I've encountered a problem while trying to install `python-xgboost-git`. I get
```
$ LC_ALL=en_US.UTF-8 sudo aura -A python-xgboost-git
aura >>= The following are not AUR packages:
python-xgboost-git

aura >>= No valid packages specified.
```
Other packages, for example, `xgboost-git` install just fine. AURA Version 1.4.0.

**Comment by Vogel612** [comment]
did you update the package index?

**Comment by leshow** [comment]
I'm also getting packages missing from the AUR

```
~
❯ sudo aura -Akxa webkitgtk2
aura >>= The following are not AUR packages:
webkitgtk2

aura >>= No valid packages specified.
```

https://aur.archlinux.org/packages/webkitgtk2/

other packages are missing also

```
aura >>= Determining dependencies...
aura >>= Dependency checking failed for these reasons:
The dependency `python2-pysmi` could not be found. You may need to search for a package to satisfy it.

~
```

https://aur.archlinux.org/packages/python2-pysmi/

I'm not sure why these packages are not found.

**Comment by fosskers** [comment]
It could be that the AUR API changed - I will have to check.

**Comment by fosskers** [comment]
@leshow does this work for you?
```
aura -Ai python2-pysmi`
```

**Comment by leshow** [comment]
@fosskers yes that works here.

**Comment by fosskers** [comment]
This will be fixed in the upcoming Aura 1.5.
